game-core/src/test/java/games/strategy/triplea/delegate/WW2V3Year41Test.java

```diff
@@ -1143,12 +1129,12 @@ void testAttackDestroyerAndSubsAgainstSubAndDestroyer() {
     final List<String> steps = battle.determineStepStrings(true);
     assertEquals(
         List.of(
-                attacker + SUBS_FIRE,
-                defender + SELECT_SUB_CASUALTIES,
+                attacker + FIRST_STRIKE_UNITS_FIRE,
```

Comment:
:+1:   I like quite a bit we are making the ordering more generic and list 'first strike units' rather than assuming any/all first strike units are subs.